index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body id="main">
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <h1 id="title">Dr. Norman Borlaug</h1>
+  <p id="tribute-info">The man who saved a billion lives</p>
+
+  <div id="img-div">
+    <img id="image" src="https://c2.staticflickr.com/4/3689/10613180113_fdf7bcd316_b.jpg" alt="" width="55%">
+    <div class="page-space"></div>
+    <caption id="img-caption">This is a beautifully taken photo!</caption>
+    <div class="page-space"></div>
+  </div>
```

Comment:
- [ ] **[REQUIRED CHANGES]** Try to always wrap up your code into `sections`, breaking it down into smaller pieces - this allows other programmers to better understand your code because you would be following the [Semantics for HTML](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html)!
It also optimizes your page SEO.

Reply:
Using sections is important for semantics and it is great that you pointed this out for the developer. However, I would suggest that you make the feedback more polite and not command. Also, it would be better to give a reason as to why the developer should wrap it into a `section`.